src/pages/Debug/Report/DebugReportActions.tsx

```diff
@@ -20,38 +23,72 @@ type DebugReportActionsProps = {
 function DebugReportActions({reportID}: DebugReportActionsProps) {
     const {translate, datetimeToCalendarTime} = useLocalize();
     const styles = useThemeStyles();
+    const [searchValue, debouncedSearchValue, setSearchValue] = useDebouncedState('');
     const [report] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`);
-    const canUserPerformWriteAction = ReportUtils.canUserPerformWriteAction(report);
+    const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID}`);
+    const ifUserCanPerformWriteAction = canUserPerformWriteAction(report);
```

Comment:
Any reason why we shouldn't just use the same term? 
```suggestion
    const canUserPerformWriteAction = canUserPerformWriteAction(report);
```

Reply:
@puneetlath it gives lint error
![image](https://github.com/user-attachments/assets/65ac214a-890c-456d-9e76-2eaf44ed6c42)